owncloudApp/src/main/res/layout/account_item.xml

```diff
@@ -80,6 +80,23 @@
         app:layout_constraintStart_toEndOf="@id/ticker"
         app:layout_constraintTop_toBottomOf="@id/name" />
 
+    <ImageView
+        android:id="@+id/cleanButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_vertical"
+        android:background="?android:attr/selectableItemBackground"
+        android:clickable="true"
+        android:paddingLeft="@dimen/standard_half_padding"
+        android:paddingTop="@dimen/standard_padding"
+        android:paddingRight="@dimen/standard_half_padding"
```

Comment:
We shouldn't use `paddingLeft` or `paddingRight`, use `paddingStart` and `paddingEnd` instead